assets/javascripts/kitten/components/uploaders/document-manager/index.js

```diff
@@ -84,6 +85,9 @@ const StyledDocumentUploader = styled.div`
     flex: 0 0 auto;
     margin-right: ${pxToRem(20)};
     align-self: flex-start;
+    @media (max-width: ${pxToRem(ScreenConfig.S.min)}) {
```

Comment:
Si on veut être pointilleux d'1px : 
```suggestion
    @media (max-width: ${pxToRem(ScreenConfig.XS.max)}) {
```